### PR TITLE
symfony 4 compatibility

### PR DESCRIPTION
**Pull request by dimonchuk**
symfony 4 compatibility

### CODE REVIEW

**Comment by dimonchuk** [comment]
Thanks @jcchavezs . Done. Please check.

**Comment by jcchavezs** [comment]
Great!

@dimonchuk could you please also add it to the circleci job for `symfony 4`?

https://github.com/jcchavezs/zipkin-instrumentation-symfony/blob/master/.circleci/config.yml#L4

**Comment by dimonchuk** [comment]
Hello @jcchavezs . Done. The test failed because in file build.js we have branch "dev-master". Should I switch to my branch? On my local machine, the test worked successfully with branch "symfony-4-compatibility".

**Comment by jcchavezs** [comment]
@dimonchuk I think you can either use `CIRCLE_BRANCH` or `*` instead of `dev-master`.

File: tests/Integration/build.sh [hunk 1]
```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env bash
 
 APP_FOLDER=test-app
-SYMFONY_VERSION=3.3
+SYMFONY_VERSION=$1
```
Comment (jcchavezs) [inline tests/Integration/build.sh]: Could we pass a default value for this as we are not informing that this is required?

File: .circleci/config.yml [hunk 2]
```diff
@@ -38,3 +69,4 @@ workflows:
   build:
     jobs:
       - "symfony-3.3"
+      - "symfony-4.0"
```
Comment (jcchavezs) [inline .circleci/config.yml]: 🏅

**Comment by jcchavezs** [comment]
@dimonchuk https://github.com/jcchavezs/zipkin-instrumentation-symfony/pull/22/files#diff-cd2866660e2555dbc3e87f0ecb77a9f8L18 fixes the branch problem. Once you fix the default value for `SYMFONY_VERSION` in `build.sh` we are ready to merge!

**Comment by dimonchuk** [comment]
Please let me know if it's not correct behavior and I'll fix. Thanks.

**Comment by jcchavezs** [comment]
@dimonchuk I just released a new version: https://github.com/jcchavezs/zipkin-instrumentation-symfony/releases

**Comment by dimonchuk** [comment]
Thank you so much!

**Pull request merged**